core/plannernotes.c

```diff
@@ -265,28 +293,54 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 					len += snprintf(buffer + len, sz_buffer - len, "<td style='padding-left: 10px; float: right;'>%s</td>", temp);
 				}
 
+				if (istrimix) {
+					runtime = (dp->time + 30) / 60;
+					deltaN2 = ((1000 - newgasmix.he.permille - newgasmix.o2.permille) - (1000 - lastprintgasmix.he.permille - lastprintgasmix.o2.permille)) / 10.0;
+					deltaHe = (newgasmix.he.permille - lastprintgasmix.he.permille) / 10.0;
+					maxdeltaN2 = 0.02 * (lastprintgasmix.he.permille-newgasmix.he.permille);
```

Comment:
Instead of multiplying by 0.02, you could compare to 50* the other value and do all this with integers.